src/CMakeLists.txt

```diff
@@ -130,10 +103,6 @@ set (NN_SOURCES
     utils/sleep.c
     utils/thread.h
     utils/thread.c
-    utils/thread_posix.h
```

Comment:
Lets talk about the changes in this file -- as well as poller.inc -- previously we had this conditional logic in macros in the source files, but now you're moving it to the cmake build.  I'm not sure if that's an excellent idea.